tests/plugins/balloonpanel/manual/positioning.html

```diff
@@ -0,0 +1,74 @@
+<h2>Divarea editor:</h2>
```

Comment:
This test is fine, however it relies on a fact that bender already has "relative body". I would declare it explicitly here so in case of something changes it still will be valid. Plus you can see what is tested on the first glimpse.

Reply:
You can also add some `margin-top`.